x-pack/legacy/plugins/apm/e2e/ci/prepare-kibana.sh

```diff
@@ -1,21 +1,13 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
-E2E_DIR="x-pack/legacy/plugins/apm/e2e"
-
-echo "1/3 Install dependencies ..."
+E2E_DIR=x-pack/legacy/plugins/apm/e2e
+echo "1/2 Install dependencies ..."
 # shellcheck disable=SC1091
 source src/dev/ci_setup/setup_env.sh true
-yarn kbn bootstrap
-
-echo "2/3 Ingest test data ..."
-pushd ${E2E_DIR}
-yarn install
-curl --silent https://storage.googleapis.com/apm-ui-e2e-static-data/events.json --output ingest-data/events.json
-node ingest-data/replay.js --server-url http://localhost:8201 --secret-token abcd --events ./events.json > ingest-data.log
+yarn kbn clean && yarn kbn bootstrap
 
-echo "3/3 Start Kibana ..."
-popd
+echo "2/2 Start Kibana ..."
 ## Might help to avoid FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 export NODE_OPTIONS="--max-old-space-size=4096"
-nohup node scripts/kibana --config "${E2E_DIR}/ci/kibana.e2e.yml" --no-base-path --optimize.watch=false> kibana.log 2>&1 &
+nohup node ./scripts/kibana --no-base-path --no-watch --dev --no-dev-config --config ${E2E_DIR}/ci/kibana.e2e.yml > ${E2E_DIR}/kibana.log 2>&1 &
```

Comment:
I do not like to launch a background process on an Agent you would have weird and inexplicable behaviors.

Reply:
well, there is a requirement to have a kibana instance up and running for that particular source code, what are the alternatives to do it?

Reply:
make the provisioning on the docker-compose, let's open a follow up to add a flag to the `compose.py` to run Kibana from sources and try to make the e2e script works with it.

Reply:
I've reopen this one https://github.com/elastic/apm-integration-testing/issues/661